demos/src/css/chartAuthoring-demo.css

```diff
@@ -1,33 +1,183 @@
-.floec-ca-pieChart-pie {
+.main {
+    width: 100%;
+    font-family: Verdana, Arial, sans-serif;
+}
+
+/* Main container */
+
+.floe-chartAuthoring-container {
+    width: 55em;
+    margin: 2em auto 2em auto;
+    border: 1px solid black;
+    padding: 1em 0 1em 0;
+    background-color: #FFFFFF;
+}
+
+/* Title and description inline edit containers */
+
+.floec-chartTitle {
+    margin-bottom: 1em;
+    padding-left: 2em;
+}
+
+.floec-chartDescription {
+    font-size: 1em;
+    margin-bottom: 2em;
+    padding-left: 2em;
+}
+
+/* Inline edit component appearance customizations */
+
+.fl-inlineEdit-text {
+    background-image:url('../img/inline_edit_edit_button_16x16.png');
+}
+
+.floec-chartTitle .fl-inlineEdit-text {
+    font-size: 1.7em;
+}
+
+.fl-inlineEdit-tooltip {
+    /* */
+}
+
+.fl-inlineEdit-editModeInstruction {
+    /* */
+}
+
+/* Pie */
+
+.floe-ca-pieChart-pie {
     float: left;
     width: 30em;
+    margin-bottom: 1em;
+}
+
+/* Pie slice text (SVG styling attributes) */
+
+.floe-ca-pieChart-text {
+    paint-order: stroke;
+    stroke: #FFF;
+    fill: #000;
+    stroke-width: 3px;
+    stroke-linecap: butt;
+    stroke-linejoin: miter;
+    font-weight: 800;
+}
+
+/* Pie slice (SVG styling attributes) */
+
+.floe-ca-pieChart-slice {
+    stroke: #FFF;
+    stroke-width: 2px;
 }
-.floec-ca-pieChart-legend {
+
+/* Legend */
+
+.floe-ca-pieChart-legend {
     float: left;
     width: 20em;
     margin: 2em;
     min-height: 5em;
 }
-.floe-ca-pieChart-legend-table {
-    border: 1px solid black;
-}
+
 .floe-ca-pieChart-legend-table thead th {
-    border: 1px solid black;
-    padding: 2px;
+    color: #FFFFFF;
 }
+
+
+/* Set to get "box" effect for legend color key cells */
+.floe-ca-pieChart-legend-color-header {
+    max-width:2.5em;
+    min-width:2.5em;
```

Comment:
Table layout in CSS is weird, and `width: 2.5em` isn't respected to limit the column width, even with overflow: hidden. Looks like max-width by itself is OK, though.